openc3/python/openc3/microservices/interface_microservice.py

```diff
@@ -218,7 +218,7 @@ def process_cmd(self, topic, msg_id, msg_hash, redis):
                 else:
                     raise RuntimeError(f"Invalid command received:\n{msg_hash}")
                 command.received_time = datetime.now(timezone.utc)
-            except RuntimeError as error:
+            except (AttributeError, RuntimeError) as error:
```

Comment:
Try sending the ASCIICMD with extra characters in the ASCII string. It raises an error but we needed to be catching the AttributeError here for it to bubble up.